backend/dal/PIMSContext.cs

```diff
@@ -1,6597 +1,120 @@
-﻿using System;
+using System.Linq;
+using System.Text.Json;
+using Microsoft.AspNetCore.Http;
 using Microsoft.EntityFrameworkCore;
-using Microsoft.EntityFrameworkCore.Metadata;
-using Pims.Dal.Entities;
-
-#nullable disable
+using Microsoft.Extensions.Options;
+using Pims.Core.Extensions;
+using Pims.Dal.Extensions;
 
 namespace Pims.Dal
 {
-    public partial class PimsContext : DbContext
```

Comment:
The generated context from the scaffold was moved to the entities project instead to keep the compilation cleaner